src/main/g8/src/test/scala/$package$/ParallelismSpec.scala

```diff
@@ -1,7 +1,8 @@
 package $package$
 
+import cats.effect.testing.scalatest.AsyncIOSpec
```

Comment:
I believe this is a now-unused remnant of a refactor.

Reply:
nice catch!